src/extension/logger.js

```diff
@@ -0,0 +1,11 @@
+import _ from 'underscore';
+
+export async function info(msg, object) {
+  if (isVerbose()) {
```

Comment:
Because cli's logging needs are very simple.
I can switch to a more advanced underlaying implementation of the logging process later on.